Mathlib/FieldTheory/IsAlgClosed/AlgebraicClosure.lean

```diff
@@ -186,29 +182,30 @@ private theorem toStepOfLE'.succ (m n : ℕ) (h : m ≤ n) :
 def toStepOfLE (m n : ℕ) (h : m ≤ n) : Step k m →+* Step k n where
   toFun := toStepOfLE' k m n h
   map_one' := by
--- Porting note: original proof was `induction' h with n h ih; · exact Nat.leRecOn_self 1`
---                                   `rw [Nat.leRecOn_succ h, ih, RingHom.map_one]`
     induction' h with a h ih
     · exact Nat.leRecOn_self 1
-    · rw [toStepOfLE'.succ k m a h]; simp [ih]
+    · simp [toStepOfLE'.succ k m a h, ih]
   map_mul' x y := by
 -- Porting note: original proof was `induction' h with n h ih; · simp_rw [Nat.leRecOn_self]`
 --                                   `simp_rw [Nat.leRecOn_succ h, ih, RingHom.map_mul]`
+    simp only
```

Comment:
Would you like to keep the porting notes here? I can see them being useful to nerd-snipe somebody into investigating, but I also think the current proof seems fine.

Reply:
Since the only difference is the extra `simp only`, I suppose this is not enough of a regression to keep around the old comment. I'll rmeove them.